swagger.json

```diff
@@ -515,11 +487,31 @@
         }
       }
     },
+    "/api/friends/getAllFriends": {
```

Comment:
nope, it's wrong. Route /api/friends/getAllFriends does not exists. While /api/fiends - return message "it's ok".

Reply:
done